scripts/deploy/28_samb_treasury_vault.ts

```diff
@@ -0,0 +1,100 @@
+import { run } from "hardhat";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import {
+  CHAINID,
+  SAMB_ADDRESS,
+} from "../../constants/constants";
+import {
+  AUCTION_DURATION,
+  PERFORMANCE_FEE,
+  PREMIUM_DISCOUNT,
+  ONE_ADDRESS,
+} from "../utils/constants";
+
+const main = async ({
+  network,
+  deployments,
+  ethers,
+  getNamedAccounts,
+}: HardhatRuntimeEnvironment) => {
+  const { BigNumber } = ethers;
+  const { parseEther } = ethers.utils;
+  const { deploy } = deployments;
+  const { deployer, owner, keeper, admin, feeRecipient } =
+    await getNamedAccounts();
+  console.log(`28 - Deploying SAMB Treasury Vault on ${network.name}`);
+
+  const chainId = network.config.chainId;
+  if (chainId !== CHAINID.ETH_MAINNET) {
+    console.log(`Error: chainId ${chainId} not supported`);
+    return;
+  }
+
+  const logicDeployment = await deployments.get("RibbonTreasuryVaultLogic");
+  const lifecycle = await deploy("VaultLifecycleTreasuryBare", {
+    contract: "VaultLifecycleTreasuryBare",
+    from: deployer,
+  });
+  console.log(`VaultLifeCycleTreasuryBare @ ${lifecycle.address}`);
+
+  const RibbonTreasuryVault = await ethers.getContractFactory(
+    "RibbonTreasuryVault",
+    {
+      libraries: {
+        VaultLifecycleTreasury: lifecycle.address,
```

Comment:
lets rename to `VaultLifecycleTreasuryBare: lifecycle.address`

Reply:
Addressed in [e34465b](https://github.com/ribbon-finance/ribbon-v2/pull/293/commits/e34465b30162c78b1c9313d3468d42cd1c325bc4)